### DISCUSSION TITLE
Customise collections sidebar

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#4263] [Feature Requests]
You can customize the collections left navigation bar right now on a per-role basis in the role settings, but not the left sidebar. Portals can be used to add new options to the right sidebar for specific layouts, but it is not possible to hide existing options (eg. information, revisions, export, layout options). Hiding them can be beneficial (eg. when a role has no access to revisions).

**Comment by rijkvanzanten** [comment]
This could be an interesting use case of https://github.com/directus/directus/discussions/3841
